rpcs3/rpcs3qt/debugger_list.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "breakpoint_handler.h"
+#include "gui_settings.h"
+
+#include "Emu/CPU/CPUThread.h"
+#include "Emu/CPU/CPUDisAsm.h"
+
+#include <QListWidget>
+
+class debugger_list : public QListWidget
+{
+	Q_OBJECT
+
+public:
+	u32 m_pc;
+	u32 m_item_count;
+	bool m_no_thread_selected;
+	QColor m_color_bp;
+	QColor m_color_pc;
+	QColor m_text_color_bp;
+	QColor m_text_color_pc;
+
+Q_SIGNALS:
+	void BreakpointRequested(u32 loc);
+public:
+	debugger_list(QWidget* parent, std::shared_ptr<gui_settings> settings, breakpoint_handler* handler);
+	void UpdateCPUData(std::weak_ptr<cpu_thread> cpu, std::shared_ptr<CPUDisAsm> disasm);
+public Q_SLOTS:
+	void ShowAddress(u32 addr);
+protected:
+	void keyPressEvent(QKeyEvent* event);
+	void mouseDoubleClickEvent(QMouseEvent* event);
+	void wheelEvent(QWheelEvent* event);
+	void resizeEvent(QResizeEvent* event);
```

Comment:
Not required I think (it worked in past ha), but it does signify intent better.  Otherwise people that don't know Qt wouldn't know that this is actually different than other methods.

Reply:
Screw it.  I'm going to add overrides everywhere where possible for Qt stuff that doesn't already have it.  No better time than now ha.